certified-connectors/RecordedFutureV2/apiDefinition.swagger.json

```diff
@@ -1212,44 +1211,44 @@
                 "created_from_relative": {
                   "type": "string",
                   "enum": [
-                    "-2",
-                    "-24",
-                    "-48"
```

Comment:
@johannesh-recordedfuture An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it.
can you please confirm that no users will be affected by this change?

Reply:
@Amjed-Ayoub I can confirm, this will replace the previous version with the removed enum values and the previous version has not been certified yet.